src/mjolnir/landmark_builder.cc

```diff
@@ -129,12 +131,20 @@ struct LandmarkDatabase::db_pimpl {
 
     // prep the select statement
     const char* select =
-        "SELECT name, type, X(geom), Y(geom) FROM landmarks WHERE ST_Covers(BuildMbr(?, ?, ?, ?, 4326), geom)";
+        "SELECT id, name, type, X(geom), Y(geom) FROM landmarks WHERE ST_Covers(BuildMbr(?, ?, ?, ?, 4326), geom)";
     ret = sqlite3_prepare_v2(db, select, strlen(select), &bounding_box_stmt, NULL);
     if (ret != SQLITE_OK) {
       throw std::runtime_error("Sqlite prepared select statement error: " +
                                std::string(sqlite3_errmsg(db)));
     }
+
+    // prep the landmark getter statement
+    const char* get_landmark = "SELECT id, name, type, X(geom), Y(geom) FROM landmarks WHERE id = ?";
```

Comment:
for clarity I'd rename `get_landmark` to `get_landmark_by_id`  and everything bbox related to `get_landmarks_by_bbox`.

Reply:
you meant the function name in this case not the prepared statement right?

Reply:
both but primarily the (public) function name, right. the prepare statement and variable names could/should be changed similarly, but that's more of a nit

Reply:
I've added `get_landmarks_by_ids` to flexibly fetch multiple landmarks by ids. the statement is not prepared in advance anymore and no min/max limit of landmark number is set. maybe we should set a max number?